src/app/app.routing.ts

```diff
@@ -9,10 +9,9 @@ import { ResetPasswordComponent } from './authentication/reset-password.componen
 import { PcpolicyComponent } from './introduction/activity-2/activity-2.component';
 import { SplashscreenComponent } from './splashscreen/splashscreen.component';
 import { RegisterComponent } from './register/register.component';
-import { NavbarComponent } from './navbar/navbar.component';
```

Comment:
not required?

Reply:
Ya! It is not required as we'll never be routing to Navbar component in router outlet.

Reply:
Alright